intdataplane/int_dataplane.go

```diff
@@ -750,3 +752,39 @@ type iptablesTable interface {
 	RemoveChains([]*iptables.Chain)
 	RemoveChainByName(name string)
 }
+
+// msgStringer wraps an API message to customise how we stringify it.  For example, it truncates
+// the lists of members in the (potentially very large) IPSetsUpdate messages.
+type msgStringer struct {
+	msg interface{}
+}
+
+func (m msgStringer) String() string {
+	if log.GetLevel() < log.DebugLevel && m.msg != nil {
```

Comment:
Yes, it's the global setting.  I was thinking if you turn on DEBUG, then expand the INFO message with the full information.